gammapy/data/gti.py

```diff
@@ -116,4 +116,20 @@ def select_time(self, time_interval):
         gti : `GTI`
             Copy of the GTI table with selection applied.
         """
-        raise NotImplementedError
+        # get GTIs that fall within the time_interval
+        mask = self.time_start >= time_interval[1]
```

Comment:
Is this implementation correct? If there are multiple GTIs present and you want want to find those overlapping the selected time interval I'd rather write:

```py3
mask = self.time_stop >= time_interval[0]
mask &= self.time_start <= time_interval[1]
self.table[mask]
```

I just realized, that it's probably equivalent to what you wrote, but confused me a little because you invert the mask at the end...maybe change to the simpler implementation?

Reply:
Done